src/components/Button.js

```diff
@@ -48,6 +48,10 @@ const propTypes = {
     /** Call the onPress function when Enter key is pressed */
     pressOnEnter: PropTypes.bool,
 
+    /** The priority to assign the enter key event listener. 0 is the highest priority. */
+    enterKeyEventListenerPriority: PropTypes.number,
+
+
```

Comment:
Extra line breaker. (We really need a eslint rule for this)

Reply:
Is there somewhere where we've explicitly added this to the review guidelines?

Reply:
Because we can easily turn on https://eslint.org/docs/rules/no-multiple-empty-lines

Reply:
> Is there somewhere where we've explicitly added this to the review guidelines

Nope.

Yeah, let's add that rule.

Reply:
@roryabraham I'm gonna create a quick PR for [eslint-config-expensify](https://github.com/Expensify/eslint-config-expensify) to turn it on.